bin/build-calendar

```diff
@@ -5,6 +5,6 @@ from pathlib import Path
 
 srcdir = Path(__file__).resolve().parent / '../src'
 sys.path.insert(0, str(srcdir))
-import bin.calendar as calendar
+import bin.event_calendar as calendar
```

Comment:
Unrelated but linter was complaining about this shadowing a [builtin module](https://docs.python.org/3/library/calendar.html)